interfaces/builtin/bool_file_test.go

```diff
@@ -194,12 +190,8 @@ func (s *BoolFileInterfaceSuite) TestConnectedPlugSnippetUnusedSecuritySystems(c
 }
 
 func (s *BoolFileInterfaceSuite) TestPermanentPlugSnippetUnusedSecuritySystems(c *C) {
-	// No extra seccomp permissions for plug
-	snippet, err := s.iface.PermanentPlugSnippet(s.plug, interfaces.SecuritySecComp)
```

Comment:
Same question here as in https://github.com/snapcore/snapd/pull/2951/files#r104112068